.github/workflows/ci.yml

```
@@ -1,44 +1,67 @@
-      - name: Install test tools
-        run: |
-          pip install pytest-cov coverage
-      - name: Run tests with coverage
-        run: |
-          pytest -q --cov=code_doc_gen --cov-report=xml:coverage.xml --cov-report=term
-      - name: Upload coverage artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: coverage-${{ matrix.python-version }}
-          path: coverage.xml
+    - uses: actions/checkout@v3
```

actionlint (1.7.10)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)